rust/blockstore/src/memory/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub(crate) mod provider;
 pub(super) mod reader_writer;
 pub(crate) mod storage;
+pub use storage::Writeable;
```

Comment:
this type leaks (or at least wasn't exposed correctly before) because `.set()` on the public BlockfileWriter interface bounds `V: Writeable`